src/Fixer/Casing/LowercaseConstantsFixer.php

```diff
@@ -59,7 +102,7 @@ protected function applyFix(\SplFileInfo $file, Tokens $tokens)
                 $this->isNeighbourAccepted($tokens, $tokens->getPrevMeaningfulToken($index)) &&
                 $this->isNeighbourAccepted($tokens, $tokens->getNextMeaningfulToken($index))
             ) {
-                $tokens[$index] = new Token([$token->getId(), strtolower($token->getContent())]);
+                $tokens[$index] = new Token([$token->getId(), $fixFunction($token->getContent())]);
```

Comment:
-1 for calling lambdas like that, no way to properly run SCA on `$lambda(...)` call.

Reply:
What do you have in mind then? What do you prefer?

Reply:
I do not use anonymous function anymore.